go.mod

```diff
@@ -244,16 +244,16 @@ require (
 	go.starlark.net v0.0.0-20230525235612-a134d8f9ddca // indirect
 	go.uber.org/multierr v1.11.0 // indirect
 	go.uber.org/zap v1.27.0 // indirect
-	golang.org/x/crypto v0.24.0 // indirect
+	golang.org/x/crypto v0.30.0 // indirect
```

Comment:
I think its fixed in `0.31.0`.
Ref: https://osv.dev/vulnerability/GHSA-v778-237x-gjrc

Reply:
updated.